CHANGELOG_PENDING.md

```diff
@@ -55,6 +55,8 @@ program](https://hackerone.com/tendermint).
     ```
     - ID is unique for each request;
     - Request.ID is now optional. Notification is a Request without an ID. Previously ID="" or ID=0 were considered as notifications.
+    - [abci] \#4177 kvstore: Return `LastBlockHeight` and `LastBlockAppHash` in `Info` (@princesinha19)
```

Comment:
please move to `### IMPROVEMENTS:` section

Reply:
Done